build.gradle

```diff
@@ -27,131 +24,24 @@ mamsdkProperties.load(new FileInputStream(mamsdkPropertiesFile))
 
 ext {
     mamSdkLibraryMaven = mamsdkProperties['mamSdkLibraryMaven']
-    if (mamSdkLibraryMaven.isEmpty()) {
```

Comment:
We don't want to do this anymore?

Reply:
Docs doesn't have this if checks, so I remove it. I can add it to the docs if needed?

Reply:
Ok. I added these error handling both in the docs and sampleapp now.